app/js/controller/list_controller.js

```diff
@@ -3,21 +3,33 @@ import { Controller } from 'components/fxos-mvc/dist/mvc';
 import ListModel from 'js/model/list_model';
 import ListView from 'js/view/list_view';
 import WebServer from 'js/lib/web_server';
+import HelpController from 'js/controller/help_controller';
 
 export default class ListController extends Controller {
   constructor() {
     this.model = new ListModel();
     this.listView = new ListView();
     this.webServer = new WebServer();
+    this.helpController = new HelpController(this);
   }
 
   main() {
     this.model.getAllApps().then((allApps) => {
-      this.enableCustomizerAddOn(allApps).then(() =>
-        this.createList(allApps));
+      this.enableCustomizerAddOn(allApps).then(() => {
```

Comment:
Won't this always show the help screen every single time you start the app? We should only show it the first time and every subsequent time after, the user can just use the Help button. You can probably just use `localStorage` to set a flag indicating whether or not they've run the app to check for this.